source/api/app.py

```diff
@@ -145,7 +145,7 @@ def start_amc_transformation():
             "--timestamp_column": timestamp_column,
             "--dataset_id": dataset_id,
             "--period": period,
-            "--country_code": country_code
+            "--country_code": country_code,
```

Comment:
Did you accidentally merge code from the "country code" feature into this branch?

Reply:
Its pre-commit change for sonarcube

Reply:
i re-merge development to this branch, so this code is outdated.